Sources/Stencil/AssetsCatalogContext.swift

```diff
@@ -28,9 +28,21 @@ extension AssetsCatalogParser {
     let catalogs = self.catalogs
       .sorted { lhs, rhs in lhs.name < rhs.name }
       .map { catalog -> [String: Any] in
+        let entries = catalog.entries.sorted { lhs, rhs in
+            if case let .group(lhsName, _) = lhs, case let .group(rhsName, _) = rhs {
```

Comment:
What do you think about implementing this using a `switch (lhs, rhs)` instead? I think it would be nicer than multiple `if case let` (which are generally better only when you switch over one case, not many like here)